src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,4 @@
+Microsoft.CodeAnalysis.VisualBasic.LanguageVersion.VisualBasic16 = 1600 -> Microsoft.CodeAnalysis.VisualBasic.LanguageVersion
```

Comment:
>1600 [](start = 67, length = 4)

Tagging @gafter @jaredpar as FYI for public API change (adding new LanguageVersion to VB)